webapp/utils/format.ts

```diff
@@ -1,14 +1,17 @@
 import { type Account } from 'btc-wallet/unisat'
 import { shorten } from 'crypto-shortener'
 import { smartRound } from 'smart-round'
-import { type Address } from 'viem'
+import { type Address, type Hash } from 'viem'
 
 export const formatBtcAddress = (account: Account) =>
   shorten(account, { length: 5 })
 
 export const formatEvmAddress = (address: Address) =>
   shorten(address, { length: 4, prefixes: ['0x'] })
 
+export const formatTxHash = (txHash: Hash) =>
+  shorten(txHash, { length: 4, prefixes: ['0x'] })
+
```

Comment:
Can we also update [here](https://github.com/hemilabs/ui-monorepo/blob/575a66bf1f55bb3fb5b497d7dae33ac13718f7b3/webapp/app/%5Blocale%5D/tunnel/transaction-history/_components/txLink.tsx#L14) to use this formatter you've created? It's the TX link in the TX history table